### PR TITLE
Ankit | Test | Version 3 - plan permission

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwm1bc1

### CODE REVIEW

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 1]
```diff
@@ -228,6 +229,9 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
     }
     /** Holds Store Get Billing Details observable*/
     public getBillingDetails$: Observable<any> = this.changeBillingComponentStore.select(state => state.getBillingDetails);
+    /** Holds View Subscription list observable*/
+    public viewSubscriptionData$ = this.ViewSubscriptionComponentStore.select(state => state.viewSubscription);
+    public remainingUsers: number = 0;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/add-company/add-company.component.ts:235]: add documentation

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts [hunk 2]
```diff
@@ -86,6 +86,8 @@ export class InvoiceSettingComponent implements OnInit, OnDestroy {
     public selectedTabIndex: number = 0;
     /** Active tab name */
     public activeTab: string;
+    /** Active company details */
+    public selectedCompany: any = null;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts]:
```suggestion
    public activeCompany: any = null;
```

Please make changes in overall file

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 3]
```diff
@@ -1450,8 +1476,9 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
      * @memberof AddCompanyComponent
      */
     public setOwnerPermission(event: any): void {
-        const isSuperAdmin = Boolean(event?.value) === true;
+        const isSuperAdmin = event?.value === "true";
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/add-company/add-company.component.ts:1479]: please check

File: apps/web-giddh/src/app/gst/filing/header/filing-header.component.ts [hunk 4]
```diff
@@ -93,6 +95,8 @@ export class FilingHeaderComponent implements OnInit, OnChanges, OnDestroy {
     public visibleSelectMonth: string = '';
     /** Instance of dayjs */
     public dayjs = dayjs;
+    /** Active company details */
+    public isActiveCompany: any = null;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/gst/filing/header/filing-header.component.ts]:
```suggestion
    public activeCompany: any = null;
```

File: apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts [hunk 5]
```diff
@@ -111,6 +111,9 @@ export class SettingPermissionComponent implements OnInit, OnDestroy {
         if (assignRoleEntryUniqueName) {
             this.store.dispatch(this.accountsAction.unShareEntity(assignRoleEntryUniqueName, 'company', this.selectedCompanyUniqueName));
             this.waitAndReloadCompany();
+            setTimeout(() => {
+                this.store.dispatch(this.settingsProfileActions.GetProfileInfo());
+            }, 1000);
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts]:
```suggestion
            }, 500);
```

File: apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts [hunk 6]
```diff
@@ -141,6 +144,9 @@ export class SettingPermissionComponent implements OnInit, OnDestroy {
     public waitAndReloadCompany() {
         setTimeout(() => {
             this.store.dispatch(this.settingsPermissionActions.GetUsersWithPermissions(this.selectedCompanyUniqueName));
+            setTimeout(() => {
+                this.store.dispatch(this.settingsProfileActions.GetProfileInfo());
+            }, 1000);
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts]:
```suggestion
            }, 500);
```

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/add-company/add-company.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/add-company/add-company.component.ts:235]: Done

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts]: Done

File: apps/web-giddh/src/app/gst/filing/header/filing-header.component.ts (hunk 4 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/gst/filing/header/filing-header.component.ts]: Done

File: apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts (hunk 5 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts]: Done

File: apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts (hunk 6 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/settings/permissions/setting.permission.component.ts]: Done

File: apps/web-giddh/src/app/add-company/add-company.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/add-company/add-company.component.ts:1479]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**